code/game/objects/items/storage/boxes.dm

```diff
@@ -476,45 +405,85 @@
 		isopened = 1
 		icon_state = "mealpackopened"
 
-//Fillable Ammo Box
-/obj/item/storage/box/ammo
-	name = "\improper Ammo Box"
-	desc = "A large ammo box that can be filled with almost any magazine type. Compact and deployable for moving ammo to the front lines."
-	icon_state = "ammobox"
+/** Fillable box
+ *
```

Comment:
Non binding request but you can use markdown for these like
/**
  * # fillable box